src/delaunay_core/math.rs

```diff
@@ -35,12 +36,14 @@ pub enum InsertionError {
     NAN,
 }
 
+#[cfg(feature = "std")]
 impl Display for InsertionError {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        <Self as std::fmt::Debug>::fmt(self, f)
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        <Self as core::fmt::Debug>::fmt(self, f)
     }
 }
 
+#[cfg(feature = "std")]
 impl Error for InsertionError {}
```

Comment:
@Stoeoef unfortunately this is one thing that actually uses std. I don't know if we can safely skip this without doing a breaking change. (yes, its annoying but Error trait is not in core nor alloc :/)
Should I remove this line? Or should we keep std as a feature for this only? Please also note that any `println!` require std (which I see commented now). But it shouldn't be an issue to use std in tests for example.

Reply:
Ugh, I didn't expect that. Removing that `impl` is a breaking change - we'll need to make it a default feature then and use `default = ["std"]`.
I probably will make std not a default feature in the next major release... too sad!

(For reference: Here's a discussion on this topic: https://github.com/rust-lang/rust/issues/62502 )